src/pages/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -23,11 +23,11 @@ exports[`Home page renders correctly 1`] = `
       >
         <a
           class="downloadButton"
-          href="https://nodejs.org/dist/v14.0.0/node-v14.0.0.tar.gz"
+          href="https://nodejs.org/dist/v12.0.0/node-v12.0.0.tar.gz"
         >
           Download Node (LTS)
           <span>
-            Version v14.0.0
+            Version v12.0.0
```

Comment:
v12 is EOL, not the LTS release

Reply:
Ah. I modified the sample data for to achieve that, looks like I misunderstood the date gaps. I'll rework it!